extensions/interactions/ItemSelectionInput/ItemSelectionInput.js

```diff
@@ -27,30 +27,56 @@ oppia.directive('oppiaInteractiveItemSelectionInput', [
       scope: {},
       templateUrl: 'interaction/ItemSelectionInput',
       controller: ['$scope', '$attrs', function($scope, $attrs) {
-        $scope.items = oppiaHtmlEscaper.escapedJsonToObj($attrs.itemsWithValue);
-        var allowableSelections = $attrs.selectionCountWithValue;
+        $scope.choices = oppiaHtmlEscaper.escapedJsonToObj($attrs.choicesWithValue);
+        var maxAllowableSelectionCount = $attrs.maxAllowableSelectionCountWithValue;
+        var minAllowableSelectionCount = $attrs.minAllowableSelectionCountWithValue;
         $scope.studentSelections = {};
 
-        for (var i = 0; i < $scope.items.length; i++) {
-          $scope.studentSelections[$scope.items[i]] = false;
+        if (maxAllowableSelectionCount == 1) {
+          $scope.isCheckbox = false;
+        } else {
+          $scope.isCheckbox = true;
+        }
+
+
+        for (var i = 0; i < $scope.choices.length; i++) {
+          $scope.studentSelections[$scope.choices[i]] = false;
         }
 
+        $scope.answers = [];
+        $scope.maxed = false; // Indicates if the count is over max.
+        $scope.min = false;  // Indicates if count is under min.
```

Comment:
$scope.notEnoughSelections

Also, put the comment in its own line above this one, and be more specific: "Indicates that the number of answers is less than minAllowableSelectionCount."